lib/rubocop/cop/mixin/shareable_constant_value.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for dealing with Shareable Constant Value
+    module ShareableConstantValue
```

Comment:
Could you please:
1) Add a comment about performance being bad
2) Move this the the cop, in a private constant? I would rather not expose this code as it is.

Reply:
Moved this to the cop.